### ISSUE TITLE
Could not find a declaration file for module 'inversify'.

### ISSUE THREAD

**Issue by Elendiar** [remojansen/InversifyJS#1622]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current behavior

After update from 6.0.2 to 6.1.2 get typescript error on `import { Container } from "inversify";` line

```
Could not find a declaration file for module 'inversify'. 'd:/user/code/sebn/project/node_modules/inversify/es/inversify.js' implicitly has an 'any' type.
  There are types at 'd:/user/code/sebn/project/node_modules/inversify/lib/inversify.d.ts', but this result could not be resolved when respecting package.json "exports". The 'inversify' library may need to update its package.json or typings.
```

My tsconfig file
```
{
  "compilerOptions": {
    "target": "ESNext",
    "useDefineForClassFields": true,
    "lib": ["ESNext", "DOM", "DOM.Iterable"],
    "types": ["reflect-metadata"],
    "module": "ESNext",
    "skipLibCheck": true,
    "plugins": [{ "name": "typescript-plugin-css-modules" }],
    "moduleResolution": "bundler",
    "allowImportingTsExtensions": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx",
    "strict": true,
    "noUnusedLocals": false,
    "noUnusedParameters": true,
    "noFallthroughCasesInSwitch": true,
    "forceConsistentCasingInFileNames": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "sourceMap": true,
    "strictNullChecks": true,
    "noErrorTruncation": true,
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"],
      "@i/*": ["src-isolation/*"]
    }
  },
  "include": ["src/**/*", "src-isolation/**/*"],
  "exclude": ["src/bindings.ts"],
  "references": [{ "path": "./tsconfig.node.json" }]
}

```

### Expected behavior

Expected type declaration

### Package version

6.1.2

### In which operating systems have you tested?

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by notaphplover** [comment]
Hey @Elendiar, thank you so much for the report. I'm on my way to fix it in #1623. I'll release a patch version after that

**Comment by notaphplover** [comment]
I just released `inversify@6.1.3` with the patch. I'll close the issue now. @Elendiar feel free to ping if the issue persist. Thank you so much for your report :smiley:

**Comment by Elendiar** [comment]
@notaphplover Thanks, the problem is gone.
